practices/development.md

```diff
@@ -2,105 +2,115 @@
-- {guilabel}`In progress`: Deliverables that we are currently working towards. This means that we should be generating [Tasks](coordination:tasks) in our [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to complete this deliverable (more on this below).
-- {guilabel}`Blocked/Waiting`: Deliverables for which we are waiting for some action that is out of our team's immediate control.
-- {guilabel}`Done`: Deliverables that have been completed. We should close these issues and celebrate the improvements that we have made!
+Monday
+: Sprint begins with our [sprint planning meeting](meetings:sprint-planning).
 
-:::{note}
-In general, there should be only two or three deliverables per team member, per column on this board.
-It should not become so heavily-populated that it is hard to keep track of deliverables!
-:::
+  In this meeting we prioritize and assign the items that each team member will work on for this week, and review items that require discussion and planning.
 
-There are two Project Backlogs at 2i2c:
+Tues-Thurs
+: Team members work on the deliverables assigned to them at the sprint planning meeting.
 
-- The [2i2c Development Projects Backlog](https://github.com/orgs/2i2c-org/projects/7?fullscreen=true) contains deliverables for our development-focused projects.
-- The [Organizational Project Backlog](https://github.com/orgs/2i2c-org/projects/8?fullscreen=true) contains deliverables that are organization-wide and not related to development.
+Friday
+: By the end of the day, team members should have completed all of their deliverables for that week.
+  Each team member fills out a status update to share the major things they worked on, and note any unexpected challenges or blockers.
 
 (coordination:deliverables)=
 ## Deliverables
 
-Deliverables represent incremental amounts of value we can deliver to a particular stakeholder, and should be completable in a week or two.
+Deliverables represent incremental amounts of value we can deliver to a particular stakeholder.
+They are **encoded as GitHub Issues**.
 
-Most issues in our repositories are deliverables, in varying states of readiness. When a deliverable is first created, it may lack information, be improperly scoped, or have an unclear path to implementation. We improve this through _deliverable refinement_ (see below).
+Most issues in our repositories are deliverables, in varying states of readiness.
+Deliverables that are added to the [Sprint Board](coordination:sprint-board) should have enough context and tasks that a team member can realistically close them on their own, and should be completable in a week.
+This may include coordinating team discussing and arriving at a decision before work can be done - in either case, the person assigned to the deliverable is responsible for carring out whatever work is needed to close it. [^invest]
```

Comment:
```suggestion
This may include coordinating team discussing and arriving at a decision before work can be done - in either case, the person assigned to the deliverable is responsible for carrying out whatever work is needed to close it. [^invest]
```